src/views/docs-view/server.ts

```diff
@@ -51,13 +51,6 @@ function getBetaLatestVersionRef(slug: string): string | undefined {
 	const hasDevPortalBranch = [
 		'vault',
 		'waypoint',
-		'terraform-docs-common',
-		'ptfe-releases',
-		'terraform-cdk',
-		'terraform',
-		'terraform-plugin-sdk',
-		'terraform-plugin-framework',
-		'terraform-docs-agents',
```

Comment:
@kylemac Just an FYI, an update here to stop reading TF content from `dev-portal` branches, per the shift last week.